PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
+## Heap Customer Support
+
+Thanks for using Heap's React Native SDK! If you're having any issues please reach out to customer support at `support@heap.io`. For the best service include "React Native" in the subject and your app id or customer email address in the body. Also, feel free to file a github issue or open a PR! If you do so, be sure to include a link in your request to customer support. Our engineering team checks for new PRs and github issues and tries to respond as soon as possible.
```

Comment:
Nit: missing comma, should be "For the best service, include ..."

Reply:
Nit: one more missing comma, should be "If you're having any issues, please reach out to ..."

Reply:
Also, we should make the email link clickable by surrounding it with angle brackets rather than ticks :-)